renderdoc/3rdparty/tinyexr/README.md

```diff
@@ -1,39 +1,43 @@
-From https://github.com/syoyo/tinyexr : 1.0
-
 # Tiny OpenEXR image library.
```

Comment:
Could you keep a note at the top of the README saying which release this is from? it's in the commit message but it's nice to have it here too to see directly.